bazel/container_structure_test.bzl

```diff
@@ -29,7 +29,9 @@ readonly yq=$(rlocation {yq_path})
 
 # When the image points to a folder, we can read the index.json file inside
 if [[ -d "{image_path}" ]]; then
-  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
+  # Never version of yq changed the default output to json and prints an annoying warning if -oj is not provided.
+  # -oj simply means (-[o]output [j]son) and -r means raw output without the quotes.
```

Comment:
nit: if the flag has a long-form, that's better to use in a script like this so it's self-documenting

Reply:
replaced -oj with long form.